src/containers/di-container.js

```diff
@@ -6,7 +7,7 @@ import GroupDataMapper from '../mappers/group-mapper';
 import UserService from '../service/user-service';
 import GroupService from '../service/group-service';
 
-const userService = new UserService(User, UserDataMapper);
+const userService = new UserService(User, UserDataMapper, jwt);
```

Comment:
just import 'jsonwebtoken' in user service, dependency injection here is not useful. Cause anyway you have strict coupling to methods that are implemented in third-party module (I mean jwt.sign), so it wont be possible to change this dependency to another one with different interface